src/python_testing/TC_WEBRTCR_2_3.py

```diff
@@ -182,9 +182,9 @@ async def test_TC_WebRTCRequestor_2_3(self):
 
             try:
                 await self.send_command("webrtc solicit-offer 3")
-                # Wait up to 30s until the provider logs that the data‑channel opened
-                if not self.th_server.event.wait(30):
-                    raise TimeoutError("DataChannel did not open within 30s")
+                # Wait up to 90s until the provider logs that the data‑channel opened
```

Comment:
Where is this 90s number coming from?  This is a certification test; is this 90s required by the spec?

Reply:
After the Offer/Answer is exchanged, the DataChannel is established

Reply:
> Based on my testing, the maximum timeout for sending a command is 30 seconds.

This _very much depends on the device_.